eclass/qt5-build.eclass

```diff
@@ -101,7 +101,16 @@ fi
 : ${QT5_BUILD_DIR:=${S}_build}
 
 LICENSE="|| ( GPL-2 GPL-3 LGPL-3 ) FDL-1.3"
-SLOT=5/$(ver_cut 1-2)
+
+case ${PN} in
+	assistant|linguist|qdbus|qdbusviewer)
+		SLOT=0 ;;
+	linguist-tools|qtdiag|qtimageformats|qtpaths|qttranslations)
```

Comment:
qtgraphicaleffects and qtquickcontrols should be here too (pure QML modules)

and what about qtplugininfo?

Reply:
qtplugininfo is treated as user-facing in Qt6, but I don't know if Qt6 plugininfo can read Qt5 metadata (and vice versa).

Reply:
> qtgraphicaleffects and qtquickcontrols should be here too (pure QML modules)

Both are installing so files although in some private subdirectory... no headers though, so I guess this is fine.

Reply:
> qtplugininfo is treated as user-facing in Qt6, but I don't know if the Qt6 tool can read Qt5 metadata (and vice versa).

Let's keep it at slot 5 then while we are unsure.

Reply:
> > qtgraphicaleffects and qtquickcontrols should be here too (pure QML modules)
> 
> Both are installing so files although in some private subdirectory... no headers though, so I guess this is fine.

yes the `.so` files just provide the (compiled C++) implementation of the QML components. By "pure QML" I meant that the modules have a QML-only API, they don't have a C++ API.

Reply:
> > qtplugininfo is treated as user-facing in Qt6, but I don't know if the Qt6 tool can read Qt5 metadata (and vice versa).
> 
> Let's keep it at slot 5 then while we are unsure.

Agreed. Should we symlink it though?